addons/project/views/project_views.xml

```diff
@@ -544,6 +544,10 @@
                                     <field name="analytic_account_id" domain="['|', ('company_id', '=', company_id), ('company_id', '=', False)]" context="{'default_partner_id': partner_id}" groups="analytic.group_analytic_accounting"/>
                                     <field name="analytic_tag_ids" groups="analytic.group_analytic_tags" widget="many2many_tags"/>
                                     <field name="privacy_visibility" widget="radio"/>
+                                    <span colspan="2" class="text-muted" attrs="{'invisible':[('access_instruction_message', '=', '')]}">
+                                        <i class="fa fa-lightbulb-o">&amp;nbsp;</i>
+                                        <field name="access_instruction_message" nolabel="1"/>  
```

Comment:
`&amp;nbsp;` is double html encoding. Once is enough in my opinion 🙂.
I would also put the white space explicitly between the `<i>` and the `<field>` html element, even if the icon is placed in the `::before` of the `<i>` as it is more "readable".
```suggestion
                                        <i class="fa fa-lightbulb-o"></i>&nbsp;<field name="access_instruction_message" nolabel="1"/>  
```